docs/reST/ref/key.rst

```diff
@@ -269,6 +269,40 @@ for ``KMOD_NONE``, which should be compared using equals ``==``). For example:
 
    .. ## pygame.key.get_pressed ##
 
+.. function:: get_just_pressed
+
+   | :sl:`returns a tuple containing the most recent key presses`
+   | :sg:`get_just_pressed() -> bools`
+
+   Returns a sequence of boolean values representing the most recent key presses.
+   When the event loop is pumped using pygame.event.get() The most updated state of the keyboard can be retrived using
+   SDL_GetKeyboardState and is used to get the most recent keydowns. Example:
+
+   ::
+      if pygame.key.get_just_pressed()[pygame.K_a]:
+         print("A key pressed")
```

Comment:
Super nitpick, but "A key pressed" is ambiguous without context. Use "b" or something else.